Documentation/Makefile

```diff
@@ -42,6 +42,7 @@ endif
 copy-api:
 	@$(ECHO_GEN)_api
 	$(QUIET)cp -r ../api _api
+	$(QUIET)rm _api/v1/docker/README.md
```

Comment:
This is a kinda quick-and-dirty hack to just fix the docs build. I don't fully understand why we're copying everything here including random go files, makefiles, etc but I no immediately obvious replacement command comes to mind so I just figured I'd add a targeted fix for the specific failure.

Also filed https://github.com/cilium/cilium/issues/13057 to figure out why we missed this in CI.